src/OrchardCore.Modules/OrchardCore.Apis.GraphQL/Startup.cs

```diff
@@ -56,7 +56,7 @@ public override void ConfigureServices(IServiceCollection services)
                 // Common types of converters are already configured in the assembly "GraphQL.SystemTextJson".
                 options.Converters.Add(GraphQLNamedQueryRequestJsonConverter.Instance);
 
-                var contentSerializerJsonOptions = sp.GetRequiredService<IOptions<ContentSerializerJsonOptions>>().Value;
+                var contentSerializerJsonOptions = sp.GetRequiredService<IOptions<DocumentJsonSerializerOptions>>().Value;
                 options.Merge(contentSerializerJsonOptions.SerializerOptions);
```

Comment:
Keeping the old variable name might be confusing.
```suggestion
                var documentJsonSerializerOptions = sp.GetRequiredService<IOptions<DocumentJsonSerializerOptions>>().Value;
                options.Merge(documentJsonSerializerOptions.SerializerOptions);
```